src/SuperDumpService/Views/Home/BundleCreated.cshtml

```diff
@@ -7,6 +7,11 @@
 @if (Model.Status == SuperDumpService.Models.BundleStatus.Failed) {
 	<h4>Bundle failed...</h4>
 	<pre>@Model.ErrorMessage</pre>
+}  else if (Model.Status == SuperDumpService.Models.BundleStatus.Duplication) {
+	<h4>This dump was already analyzed. You'll be redirected to the original dump.</h4>
```

Comment:
will this message ever be visible? IMHO it would be good to know that it was a duplicate. either make a timeout of a few seconds before redirect or just offer a plain link.